DigitalLearningSolutions.Web.Tests/Controllers/LearningMenu/LearningMenuControllerTests.cs

```diff
@@ -85,5 +85,50 @@ public void ContentViewer_should_render_view()
             result.Should().BeViewResult()
                 .Model.Should().BeEquivalentTo(expectedModel);
         }
+
+        [Test]
+        public void Index_always_calls_checks_course_content()
+        {
+            // Given
+            const int customisationId = 1;
+
+            // When
+            controller.Index(1);
+
+            // Then
+            A.CallTo(() => courseContentService.GetCourseContent(customisationId)).MustHaveHappened();
+        }
+
+        [Test]
+        public void Index_valid_customisation_id_should_call_the_course_content_service()

```

Comment:
This test name isn't quite accurate, it doesn't describe what it should call on the course content service. Something like `should_update_login_and_duration` would be more descriptive I think 🙂